tests/components/axis/test_camera.py

```diff
@@ -75,9 +70,7 @@ async def test_camera_with_stream_profile(hass, config_entry):
     )
 
 
-async def test_camera_disabled(hass, config_entry):
+@patch("axis.vapix.vapix.Params.image_format", return_value=None)
```

Comment:
I recommend not mixing pytest fixtures with patch decorators. It's easy to make a mistake.

How do we know the order of patch application vs fixture injection here? Does it matter?

Reply:
The test would fail if the patch didn't apply. I can look at fixing this in a separate PR.

Reply:
Changed